packages/panzer/dof-mgr/src/Panzer_FieldAggPattern.hpp

```diff
@@ -147,6 +148,25 @@ class FieldAggPattern : public FieldPattern {
      *                  required order of the basis functions
      */
    const std::vector<int> & localOffsets(int fieldId) const;
+
+  /** This function produces a map between the ordering specified by
+     * this pattern and an ordering required by the pattern associated with
+     * <code>fieldId</code>. 
```

Comment:
If you like, you can use `` `backticks` `` here.  Doxygen translates them to the appropriate `<code></code>` tags.